core/src/ldml/ldml_vkeys.cpp

```diff
@@ -65,6 +65,16 @@ vkeys::lookup(km_core_virtual_key vk, uint16_t modifier_state, bool &found) cons
       return ret;
     }
   }
+
+  // look for a layer with "default"
+  {
+    const vkey_id id_default(vk, (K_DEFAULTMODFLAG));
```

Comment:
```suggestion
    const vkey_id id_default(vk, (DEFAULT_MODIFIER));
```

Reply:
that constant was removed, so using LDML_KEYS_MOD_DEFAULT instead